UI/js/common.js

```diff
@@ -23,3 +23,19 @@ const defaultError = 'Something went wrong, please try again!';
 function getErrorMessageFromResponse(data) {
     return data ? errors[data['code']] || defaultError : defaultError;
 }
+
+/**
+ * Allows us to call an animate function with a callback
+ */
+$.fn.extend({
```

Comment:
Fix the format

Reply:
I think formatting is okay
not sure what you mean

Reply:
The use of var. String addition instead of interpolation. Fix those plz